doc/api/http2.md

```diff
@@ -24,8 +24,10 @@ be emitted either by client-side code or server-side code.
 
 ### Server-side example
 
-The following illustrates a simple, plain-text HTTP/2 server using the
-Core API:
+The following illustrates a simple HTTP/2 server using the Core API.
+Since there are no browsers known that support [unencrypted HTTP/2](HTTP/2 Unencrypted),
```

Comment:
1. `(HTTP/2 Unencrypted)` -> `[HTTP/2 Unencrypted]` (otherwise it is not rendered as a link).
2. This and next line exceed 80 character length, so this may be a problem for doc linting

Reply:
Whoa good catch, I just copy/pasted the code review comment so there must have been a typo.